android/src/main/java/com/bitmovin/player/reactnative/RNPlayerView.kt

```diff
@@ -165,6 +170,12 @@ class RNPlayerView(context: Context, appContext: AppContext) : ExpoView(context,
 
     fun dispose() {
         activityLifecycle?.removeObserver(activityLifecycleObserver)
+        appContext.activityProvider?.currentActivity
```

Comment:
In [this PR](https://github.com/bitmovin/bitmovin-player-react-native/pull/762) I added a dispose function to the PictureInPictureHandler. I think it might make sense to have it there as I think this code here is quite confusing without any context. Additionally ideally we should not change the activity behaviour if our PiP implementation was not used in case the app uses some other library that might have configured this on the activity.